testsuite/tests/asmgen/tagged-integr.cmm

```diff
@@ -31,7 +31,7 @@ arguments = "-DINT_FLOAT -DFUN=test main.c"
         r))
 
 (function "integr" (f: val low: val high: val n: int)
-  (let (h "h" x "x" s "s" i n)
+  (letmut (h val "h" x val "x" s val "s" i int n)
```

Comment:
Do you really need all of the variables to be mutable here ? It seems making only `i` mutable should be enough.

Reply:
It would! I just didn't bother implementing `parsecmm` syntax for a mixture of mutable and nonmutable bindings.